src/components/text/CryptoText.tsx

```diff
@@ -48,7 +47,7 @@ export const useCryptoTextSimple = ({ wallet, tokenId, nativeAmount, withSymbol
     exchangeRate: assetToFiatRate,
     fiatDenomination,
     nativeAmount,
-    currencyCode: withSymbol ? undefined : currencyCode
+    currencyCode: withSymbol ? undefined : displayDenomination.name
```

Comment:
I would have expected the field to be `displayDenomination.currencyCode` or `displayDenomination.symbol`. Are you sure this is right?